src/engine/objects/robot/SimRobotDrivetrain.ts

```diff
@@ -192,6 +192,11 @@ export class SimRobotDrivetrain {
     motor.inputSignal = value;
   }
 
+  getMotorInputSignal(channel: number): number {
+    const motor = this._motors.get(channel);
```

Comment:
If `_motors` does not have the `channel` we'll return `undefined.inputSignal` which will crash the screen I guess? But it seems already throwing a typeerror exception so the caller can just handle it.

Reply:
First, it doesn't have to crash the screen, we can check for `undefined`.
Second, we iterate over the robot configuration so we should not run into situation where we ask for channel that doesn't exists.